dqc_us_rules/dqc_us_0033_0036.py

```diff
@@ -201,7 +201,7 @@ def _get_default_dped(dped_facts):
         return None
     elif len(keys) == 1:
         # Only one Document Period End Date, so that is our default
-        value, = dped_facts.values()
+        value = dped_facts.values()
```

Comment:
why was this changed?  The ```,``` made it so that the list dereferenced.

A better fix might be to do ```value = dped_facts.values()[0]```